src/patternfly/components/Wizard/wizard-title.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="pf-c-wizard__title{{#if wizard-title--modifier}} {{wizard-title--modifier}}{{/if}}"
+  {{#if wizard-title--attribute}}
+    {{{wizard-title--attribute}}}
+  {{/if}}>
+  {{#if wizard-title-text--value}}
+    {{#> wizard-title-text}}
+      {{{wizard-title-text--value}}}
+    {{/wizard-title-text}}
+  {{else if @partial-block}}
```

Comment:
This could probably just be an `else`, right?

Reply:
Yep! I can open a followup issue for some other components that could use a similar update (Badge being one)

Reply:
@thatblindgeye I'm sorry - I've learned that it's actually better to use the `{{else if @partial-block}}` so that you can use the syntax `{{> thing}}` rather than having to use an open and close `{{#> thing}}{{/thing}}`